catalog/confluence/catalog-confluence-source/src/main/java/org/codice/ddf/confluence/source/ConfluenceSource.java

```diff
@@ -124,12 +123,18 @@ public ConfluenceSource(FilterAdapter adapter, EncryptionService encryptionServi
     }
 
     public void init() {
+        if (StringUtils.isBlank(endpointUrl)) {
```

Comment:
@figliold are you implying we should log something there? I could see adding a log to tell the administrator to configure/set an endpoint URL, but I might be overthinking that.